### DISCUSSION TITLE
Uploaded the SSH Keys and ready for taking the changes

### DISCUSSION THREAD

**Discussion by Satish-Mutyala** [Satish-Mutyala/T24S2T1EXTG12#5] [Announcements]
Hi Team, I have uploaded public keys into the github, you are all ready to close the repository and start developing the calculators features. Please make a branch off the development.

**Comment by WeiZhou-dev** [comment]
Hi Satish, thanks for letting us know. Could you please update the screenshot for Part B Task B-b to show that we have accepted the invitation? Thanks a lot.

**Comment by JeongseopYun** [comment]
Hello Satish, thank you for letting us know. I will finalise my Currency Calculator and merge it into the development branch.
